apps/api/src/app/invites/usecases/invite-member/invite-member.usecase.ts

```diff
@@ -37,7 +37,6 @@ export class InviteMember {
     if (process.env.NOVU_API_KEY && (process.env.NODE_ENV === 'dev' || process.env.NODE_ENV === 'production')) {
       const novu = new Novu(process.env.NOVU_API_KEY);
 
-      // eslint-disable-next-line @cspell/spellchecker
       // cspell:disable-next
```

Comment:
You should remove all the ocurrences of cspell anotations.

Reply:
The second annotation helps the CI skip this one. I only removed the eslint one here